basiclti/web-ifp/src/java/org/sakaiproject/portlets/SakaiIFrame.java

```diff
@@ -281,9 +281,10 @@ public void doView(RenderRequest request, RenderResponse response)
 
 		// The current user may not be a maintainer in the current site, but we want to still be able to
 		// correct the source on the LTI tool
-		Object retval = m_ltiService.insertContentDao(props, siteId, m_ltiService.isAdmin(siteId), true);
+		// Also pretend that the user is an admin so they can have stealth tools patched.
+		Object retval = m_ltiService.insertContentDao(props, siteId, true, true);
```

Comment:
We don't need to check if the user is an admin anymore?

Reply:
No, otherwise they can't save a stealthed tool in their My Workspace.